features/post/post-comment/post-comment.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactNode } from 'react';
+
+import useRelativeTime from '@/shared/hooks/useRelativeTime';
+import { useTranslation } from '@/shared/hooks/useTranslation';
+import { Avatar, Typography } from '@/shared/ui';
+import clsx from 'clsx';
+
+import s from './post-comment.module.scss';
+
+export type PostCommentT = {
+  avatar?: string;
+  children?: ReactNode;
+  childrenInfoSection?: ReactNode;
+  createdAt: string;
+  iconElement?: ReactNode;
+  likesCount: number;
+  text: string;
+  textWidth?: string;
+  userName: string;
+};
+
+export const PostComment = (props: PostCommentT) => {
+  const {
+    avatar,
+    children,
+    childrenInfoSection,
+    createdAt,
+    iconElement,
+    likesCount,
+    text,
+    textWidth,
```

Comment:
`textWidth` is a bad name for a prop representing a class name. The width implies some numeric value.

Also, this prop is used to limit the width of the entire container with the response to the comment, which is not logical. Here we need to split the username and comment text into 2 tags and limit the width of the container, not the text inside.

To be able to limit the width of the container requires a classes-API, preferably with the ability to customize classes for most of the underlying elements. Especially since this is a fully emendable component, so it should be flexible for customization. Implementation of a similar API already exists in some components (see the `classes` props).